src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -60,6 +65,40 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
   const [searchValue, setSearchValue] = useState('');
   const [sortBy, setSortBy] = useState({ index: 1, direction: SortByDirection.asc });
   const [rowExpandedStatus, setRowExpandedStatus] = useState(new Array(10).fill(false));
+  const [currentDetailUUID, setCurrentDetailUUID] = useState('');
+  const [detailsDrawerIsExpanded, setDetailsDrawerIsExpanded] = useState(false);
+  const [currentDetailRowIndex, setCurrentDetailRowIndex] = useState(null);
+
+  const drawerRef = useRef(null);
+
+  const openDetailsPanel = (uuid: string, rowIndex: number): void => {
+    setCurrentDetailUUID(uuid);
+    setCurrentDetailRowIndex(rowIndex);
+    setDetailsDrawerIsExpanded(true);
+  };
+
+  const closeDetailsPanel = () => {
+    setCurrentDetailUUID('');
+    setDetailsDrawerIsExpanded(false);
+  };
+
+  const formatRow = (row: string[], rowIndex: number) => {
+    const name: string = row[0];
+    const version = `${row[1]}`;
```

Comment:
What is this interpolation for?

Reply:
That was I think just leftover type casting; I had thought it was passing the version across as a number at one point (and wanted to be consistent) but I must have missed cleaning this up.